packages/contents/src/drivefs.ts

```diff
@@ -326,7 +326,7 @@ export class DriveFSEmscriptenNodeOps implements IEmscriptenNodeOps {
     const path = this.fs.PATH.join2(this.fs.realPath(node), name);
     const result = this.fs.API.lookup(path);
     if (!result.ok) {
-      throw this.fs.FS.genericErrors[this.fs.ERRNO_CODES['ENOENT']];
+      throw new this.fs.FS.ErrnoError(this.fs.ERRNO_CODES['ENOENT']);
```

Comment:
Neat! Thanks!

Question, was `ErrnoError` also available for old emscripten versions? Or is it a new thing?
So should we detect if `genericErrors` is defined, and use it if it is, with a fallback on `ErrnoError`?

Reply:
Nope, ErrnoError is pretty fundamental as far as I can see and has been there since quite some time.

We actually use it everywhere while throwing (apart from this block)

https://github.com/jupyterlite/jupyterlite/blob/7c67c2e5c3f4734a897652ea18c35244baf30a0f/packages/contents/src/drivefs.ts#L246

https://github.com/jupyterlite/jupyterlite/blob/7c67c2e5c3f4734a897652ea18c35244baf30a0f/packages/contents/src/drivefs.ts#L382

https://github.com/jupyterlite/jupyterlite/blob/7c67c2e5c3f4734a897652ea18c35244baf30a0f/packages/contents/src/drivefs.ts#L473